apps/example/src/app/simple-form/simple-form.component.ts

```diff
@@ -158,9 +151,9 @@ export default class SimpleFormComponent {
           ),
         };
       }),
-      todos: this.sfb.createFormGroup<WritableSignal<FormGroup<Todo>[]>>(
+      todos: this.sfb.createFormGroup<Todo[]>(
```

Comment:
While I was working on the types, I didn't like this generic and the use of `signal` so I also refactored this API.
The array will be wrapped in a signal within signal-forms, which makes it easier for the user.